.github/workflows/common.yml

```diff
@@ -132,12 +132,13 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Run Merge Gatekeeper
-        uses: upsidr/merge-gatekeeper@v1
+        uses: upsidr/merge-gatekeeper@v1.2.0
```

Comment:
Why you used full version here?

Reply:
Because they don't use the tag `latest` which is why we end up using an older version.

Reply:
Ah, that's because they have exactly `v1` version tag alongside with proper `v1.0.0`.